sdk/metric/config.go

```diff
@@ -14,12 +14,120 @@
 
 package metric // import "go.opentelemetry.io/otel/sdk/metric"
 
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"go.opentelemetry.io/otel/sdk/metric/view"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
 // config contains configuration options for a MeterProvider.
-type config struct{}
+type config struct {
+	res     *resource.Resource
+	readers map[Reader]view.Config
```

Comment:
Note: this was just chosen as a place-holder data type for a reader/view. There's no reason not to change this going forward if we determine a better data type for this.